Lib/multiprocessing/pool.py

```diff
@@ -164,7 +174,8 @@ def __init__(self, processes=None, initializer=None, initargs=(),
         self._ctx = context or get_context()
         self._setup_queues()
         self._taskqueue = queue.SimpleQueue()
-        self._cache = {}
+        self._change_notifier = self._ctx.SimpleQueue()
```

Comment:
Please add a comment to explain the purpose of this queue. I understand that it exists to wake up _handle_workers(): when _state value is changed.